### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Support for Markdown Formatting Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by GreenFlux** [dailydotdev/daily#1924] [Feature Requests]
### Concise Description

Add keyboard shortcuts for formatting the selected text as bold, italics, and inserting hyperlinks

### Description

Requesting support for Markdown keyboard shortcuts. 

### Solution Proposed

- cmd/ctrl + b: set selected text as bold
- cmd/ctrl + i: set selected text as italics
- cmd/ctrl + k: replace selected text with [text](url) placeholder
- cmd/ctrl + v: when pasting a link over selected text, convert to hyperlink

### Additional Context

```bash
Currently all Markdown formatting characters have to be typed manually. Providing a better writing experience will encourage more text posts, instead of linking to external blogs.
```

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @GreenFlux,

Thanks for this great suggestion. We will open it for public contribution :)

**Comment by Abhi-Bohora** [comment]
Hello, I will be working on this one....😊

**Comment by idoshamun** [comment]
Amazing! Top contributor 💪 
@sshanzel fyi

**Comment by sshanzel** [comment]
@Abhi-Bohora this is one of my favorite features. Good luck with it, and let me know if you need any help 🤩

**Comment by Abhi-Bohora** [comment]
@sshanzel Hello thanks for your support, I am also excited to work on this. How do you suggest that I should build this? I was thinking if you guys are deciding to use some formatting toolbar like library or something that can go together with our existing `MarkdownInput` component and maybe that library will also help here to implement different shortcut and maybe less chance to missing weird cases than while formatting and unformatting the word selected by the user manually by applying own logic. And will close this issue https://github.com/dailydotdev/daily/issues/1625 also along with this issue ?
